docs/data/sql_functions.yml

```diff
@@ -708,6 +708,41 @@ json:
       -- '[]'
       JSON_QUERY('{}', 'strict $.invalid' EMPTY ARRAY ON ERROR)
       ```
+  - sql: JSON_OBJECT([KEY key VALUE value]* [ { NULL | ABSENT } ON NULL ])
+    table: jsonObject(JsonOnNull, keyValues...)
+    description: |
+      Builds a JSON string from a list of key-value pairs.
+
+      Note that keys must be non-`NULL` string literals, while values may be arbitrary expressions.
+
+      The `ON NULL` behavior defines how to treat `NULL` values.
+
+      Note that this function only returns a JSON string, not an actual JSON type. Not all JSON can
```

Comment:
(Also, that's just how Calcite implements it, but this is the better reason to state)